textile-utils-api-v1/src/main/resources/textile-utils-api-v1.accessWidener

```diff
@@ -0,0 +1,2 @@
+accessWidener	v1	named
+accessible  method  net/minecraft/block/Block   <init>  (ILnet/minecraft/block/Material;)V
```

Comment:
Unnecessary access widener

Reply:
That AW is necessary for Pair to function.

Reply:
Really? It wasn't when I used it

Reply:
Oh yeah, I was using a custom block class

Reply:
That's because you instantiated your Block class, I had to instantiate it as a Block as I don't want to hardcode it to be "Stone" or whatnot.

Reply:
Why is a Block access widener *necessary*

Reply:
Perhaps in the future write a more extensive api that has all modded blocks be instances of FabricBlock, and thus all with same numerical id, but also have String id, which is checked every time the numerical id is checked, but that would require many mixins to fix that, and that is probably later down the rodd

Reply:
Look at the top issue. I commented about some ways to do that.